### PR TITLE
Merge videoUploading samples together

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR merges both videoUploading and videoUploadingWebCodecs samples together as there's not much value in having both when only the external texture source changes. Note that I set up a redirect so that existing links to videoUploadingWebCodecs still work properly.

See diff at https://www.diffchecker.com/FUiV227s/

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -8,6 +8,15 @@ module.exports = {
   basePath: BASE_PATH,
   compress: true,
   reactStrictMode: true,
+  async redirects() {
```
[the remark this answers is not in the document]
Comment (beaufortfrancois) [reply next.config.js:11]: I followed next.js documentation.

**Review by austinEng — APPROVED** [review]

**Comment by beaufortfrancois** [comment]
@austinEng Unlike my local environment, https://webgpu.github.io/webgpu-samples/samples/videoUploadingWebCodecs does not redirect to https://webgpu.github.io/webgpu-samples/samples/videoUploading?videoSource=videoFrame. Do you know what is happening?

**Comment by beaufortfrancois** [comment]
Could it be because I missed "/webgpu-samples/" in redirection URLs?

**Comment by beaufortfrancois** [comment]
@austinEng I've just sent https://github.com/webgpu/webgpu-samples/pull/365/ for review

**Pull request merged**